docs/cookbook/ddd_architecture/basic_operations.md

```diff
@@ -0,0 +1,462 @@
+            new BookContent($data->content),
+            new Price($data->price),
+        );
+
+        $model = $this->commandBus->dispatch($command);
+
+        return BookResource::fromModel($model);
+    }
+}
+```
+
+### Adding the processor on the Book Resource
+
+And then we create the "update" operation on our BookResource.
+
+```php
+// src/BookStore/Infrastructure/Sylius/Resource/BookResource.php
+
+// ...
+use App\BookStore\Infrastructure\Sylius\State\Processor\CreateBookProcessor;
+use App\BookStore\Infrastructure\Symfony\Form\BookResourceType;
+use Sylius\Resource\Metadata\Create;
+use Sylius\Resource\Model\ResourceInterface;
+// ...
+
+#[AsResource(
+    // ...
+    formType: BookResourceType::class, // Define the form type for all your operations
+    operations: [
+        new Create(
+            processor: CreateBookProcessor::class, // the processor we have just created
+            // formType: CreateBookResourceType::class, // Optional: define a specific form type only for the "create" operation
+        ),
+    ],
+)]
+final class BookResource implements ResourceInterface
+{
+    // ...
+}
+```
+
+## Book edition
+
+<div data-full-width="false">
+
+<figure><img src="../../.gitbook/assets/update_book_resource.png" alt="Update book resource"></figure>
+
+</div>
+
+In the Application folder, we already have this `UpdateBookCommand`:
+
+```php
+// src/Bookstore/Application/Command/UpdateBookCommand.php
+
+declare(strict_types=1);
+
+namespace App\BookStore\Application\Command;
+
+use App\BookStore\Domain\Model\Book;
+use App\BookStore\Domain\ValueObject\Author;
+use App\BookStore\Domain\ValueObject\BookContent;
+use App\BookStore\Domain\ValueObject\BookDescription;
+use App\BookStore\Domain\ValueObject\BookId;
+use App\BookStore\Domain\ValueObject\BookName;
+use App\BookStore\Domain\ValueObject\Price;
+use App\Shared\Application\Command\CommandInterface;
+
+/**
+ * @implements CommandInterface<Book>
+ */
+final readonly class UpdateBookCommand implements CommandInterface
+{
+    public function __construct(
+        public BookId $id,
+        public ?BookName $name = null,
+        public ?BookDescription $description = null,
+        public ?Author $author = null,
+        public ?BookContent $content = null,
+        public ?Price $price = null,
+    ) {
+    }
+}
+```
+
+In the same folder, we also have this existing `FindBookQuery`:
+
+```php
+// src/BookStore/Application/Query/FindBookQuery.php
+
+declare(strict_types=1);
+
+namespace App\BookStore\Application\Query;
+
+use App\BookStore\Domain\Model\Book;
+use App\BookStore\Domain\ValueObject\BookId;
+use App\Shared\Application\Query\QueryInterface;
+
+/**
+ * @implements QueryInterface<Book>
+ */
+final readonly class FindBookQuery implements QueryInterface
+{
+    public function __construct(
+        public BookId $id,
+    ) {
+    }
+}
+```
+
+The idea is to reuse these query and command to update the book in the storage for your "update" operation.
```

Comment:
Same comment as above, I'm not sure how but I think this sentence should be rephrased to be less repetitive